crates/rethnet_eth/src/remote/client.rs

```diff
@@ -698,8 +812,8 @@ mod tests {
     }
 
     #[tokio::test]
-    async fn send_request_body_500_status() {
-        const STATUS_CODE: u16 = 500;
+    async fn send_request_body_400_status() {
```

Comment:
Why 400?

Reply:
If I understand correctly, this test was a smoke test that things don't blow up when the server returns an http error code. We're retrying 5xx requests now, so we can't test that here. I picked 400, because it's a common error that isn't retried.